### PR TITLE
Suspend/Resume

### PR DESCRIPTION
**Pull request by ps2**
This applies on top of `initial-implementation` branch. Once that branch has landed in dev, I'll rebase this to dev.

https://tidepool.atlassian.net/browse/LOOP-356

### CODE REVIEW

File: DashKitUI/ViewControllers/DashSettingsViewController.swift [hunk 1]
```diff
@@ -148,7 +167,7 @@ class DashSettingsViewController: UITableViewController {
 
     override func tableView(_ tableView: UITableView, shouldHighlightRowAt indexPath: IndexPath) -> Bool {
         switch Section(rawValue: indexPath.section)! {
-        case .replacePod:
+        case .actions:
```
Comment (darinkrauss) [inline DashKitUI/ViewControllers/DashSettingsViewController.swift]: Intentionally that `.replacePod` now returns `false`?

**Review by darinkrauss — APPROVED** [review]
LGTM. Only one question about whether a change was intentional or not. If it was, then 🚀.

**Review by nhamming — APPROVED** [review]
LGTM

File: DashKitUI/ViewControllers/DashSettingsViewController.swift (hunk 1 above)
Comment (ps2) [reply DashKitUI/ViewControllers/DashSettingsViewController.swift]: Ah, no. Good catch.

**Review by darinkrauss — APPROVED** [review]
LGTM! 🚀

**Pull request merged**